Woof/Woof/View/OwnerFlow/DetailedPetSitterView/DetailedPetSitterView.swift

```diff
@@ -13,10 +13,26 @@ struct DetailPetSitterView: View {
                 iconName: .IconName.filledPerson,
                 text: viewModel.fullName
             )
+            .contextMenu {
+                Button {
+                    viewModel.copyToClipboardText(viewModel.fullName)
+                } label: {
+                    CopyToClipboardLabel()
+                }
+            }
+
             TextWithIconLabelView(
                 iconName: .IconName.phone,
                 text: viewModel.phoneNumber
             )
+            .contextMenu {
+                Button {
+                    viewModel.copyToClipboardText(viewModel.phoneNumber)
+                } label: {
+                    CopyToClipboardLabel()
+                }
+            }
```

Comment:
nit:
Can we have reusable view that handles the common functionality to avoid DRY?

And then use it like this:
```suggestion
CopyableTextWithIconLabelView(
                iconName: .IconName.phone,
                text: viewModel.phoneNumber
)
```

Reply:
We have to open context menu and pass the text from view model. I have not obvious decision how to implement you suggestion.

Reply:
Okay, it just a nit. 
Please, think how you can avoid DRY for the next time.